app/adapters/cron.js

```diff
@@ -0,0 +1,44 @@
+import Ember from 'ember';
+import ApplicationAdapter from 'travis/adapters/application';
+
+export default ApplicationAdapter.extend({
+
+  findRecord(store, type, id, record) {
+    return this.ajax(this.urlPrefix() + '/cron/' + id, "GET");
```

Comment:
I think that should be the default behaviour for adapter (and the same goes for all the other methods here), have you tried to run the code without specifying these?